build.gradle

```diff
@@ -29,6 +30,10 @@ dependencies {
     implementation 'com.fasterxml.jackson.core:jackson-databind:2.11.2'
     implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72'
     implementation 'org.jetbrains.kotlin:kotlin-reflect:1.3.72'
+    implementation 'io.shiftleft:codepropertygraph_2.13:1.2.26'
+
+    implementation project(":cpgconv")
```

Comment:
A multi-project build was the only viable option here. I first tried to simply have `kotlinCompile` depend on `scalaCompile`, but that creates a cyclic dependency.